pyInterface/test_MC.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+### BEGIN CONFIGURATION ###
+
+	### BEGIN PHASE SPACE ###
+
+		# mass bin (MeV)
+		MASS=1800
+		BINWIDTH=20
+
+		# number of phase space events to generate 
+		NMB_PS_EVENTS=500	
+
+	### END PHASE SPACE ###
+
+	### BEGIN PARTICLE DATA ###
+
+		# particle data table
+		PARTICLE_DATA_TABLE="${ROOTPWA}/particleData/particleDataTable.txt"
+
+	### END PARTICLE DATA ###
+
+	### BEGIN PSEUDO DATA ###
+
+		# number of weighted pseudo data events to generate
+		NMB_PSEUDO_EVENTS=500
+
+	### END PSEUDO DATA ###
+
+
+#-- END CONFIGURATION --#
+
+
+### BEGIN PREPARATION ###
+
+if [ -d "${ROOTPWA}" ]; then
+	echo "RootPwa installation found in: ${ROOTPWA}"
+	echo "Starting MC test ..."
+else
+	echo "Error no installation of RootPwa found! Aborting."
+	exit 1
+fi
+
+TESTDIR=${PWD}/ROOTPWA_MC_TEST
+
+if [ -d "${TESTDIR}" ]; then
+	echo "Test directory ${TESTDIR} already exists! Aborting."
+	exit 1
+fi
+
+echo "Creating directories ..."
+
+mkdir "${TESTDIR}"
+mkdir "${TESTDIR}/data"
+mkdir "${TESTDIR}/weighted_mc_data"
+mkdir "${TESTDIR}/amps"
+mkdir "${TESTDIR}/ints"
```

Comment:
not manually creating `amps/` and `ints/` would test that `createFileManager.py` indeed creates these directories (at least the current version should do this)